apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/anonymous/AEPSampleAppNewArchEnabled/MainActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.anonymous.AEPSampleAppNewArchEnabled
+
```

Comment:
The license header is required for all javascript files and Kotlin files, such as https://github.com/adobe/aepsdk-react-native/blob/main/apps/AEPSampleApp/android/app/src/main/java/com/aepsampleapp/MainActivity.kt#L1-L10

Reply:
was not aware of it . have added now